usaspending_api/common/query_with_filters.py

```diff
@@ -312,7 +312,7 @@ def _generate_elasticsearch_query(cls, filter_values: List[str], query_type: _Qu
         naics_codes_query = []
 
         for v in filter_values:
-            naics_codes_query.append(ES_Q("match", naics_code__keyword=v))
+            naics_codes_query.append(ES_Q("wildcard", naics_code__keyword=f"{v}*"))
```

Comment:
+1 for changing the `match` and `match_phrase_prefix` to `wildcard`, and being able to take advantage of the `naics_code.keyword` subfield. It should make the ES impl faster.